static/compiler.js

```diff
@@ -752,6 +755,9 @@ define(function (require) {
         this.saveState();
         this.compile();
         this.updateButtons();
+        if(this.getEffectiveFilters().binary){
+            this.eventHub.emit('binaryMode');
```

Comment:
Pass the compiler id for which the binaryMode was toggled and its new state so not every cfg from every compiler gets updated when only one does